src/metrics/index.md

```diff
@@ -0,0 +1,53 @@
+# Standard Metrics
+
+## Activity
+
+### DAU
+
+The number of unique profiles active on each day.
+
+### WAU
+
+The number of unique profiles active at least once during the 7-day window
+ending on the specified day.
+
+### MAU
+
+The number of unique profiles active at least once during the 28-day window
+ending on the specified day.
+
+### Intensity
+
+Intuitively, how many days per week do users use the product? Among profiles
+active at least once in the week ending on the date specified, the number of
+days on average they were active during that one-week window.
+
+## Retention
+
+### 1-Week New Profile Retention
+
+Among new profiles created on the day specified, what proportion (out of 1) are
+active during the week beginning one week after the day specified.
+
+### 1-Week Retention
+
+Among profiles that were active at least once in the week starting on the
+specified day, what proportion (out of 1) are active during the following week.
+
+## Frequently Asked Questions
```

Comment:
I'm concerned these are too prominent. Can we add them to the GUD article?

Since these are in *Getting Started* I feel like I need to understand why these questions are important for all new analysts to see.

Reply:
I would argue that these particular things are important enough to be read by everyone when getting started. They're not super obvious, and (other than the first one) have been subject to debate / change in the past.